src/prommis/nanofiltration/costing/diafiltration.py

```diff
@@ -0,0 +1,713 @@
+#########################################################
+# MSContactor flowsheet code adapted from Andrew Lee.
+#########################################################
+
+
+import pyomo.environ as pyo
```

Comment:
This is generally discouraged for core code - it is better to only import the components you will actually use (this helps with maintenance too). This type of import is OK for simpler user facing examples and demos to cut down on time and explanation however.